setup-libraries.ipynb

```diff
@@ -113,7 +348,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython2",
-   "version": "2.7.14+"
+   "version": "2.7.5"
```

Comment:
You're clearly not using CMSSW_10_6_13 or this would say 2.7.14+

Reply:
I encountered this in my own setup, too, and was using CMSSW_10_6_13. Not sure why. What does this parameter control?

Reply:
It's not truly important, but it means that we're not all testing with consistent environments.